src/main/java/bit/javaoop/Employee.java

```diff
@@ -9,6 +9,7 @@ public Employee(String name, double salary) {
 
     @Override
     public boolean isSatisfied() {
-        return getSalary() > 10_000;
+        return getSalary() > 10000;
```

Comment:
Tutaj te magiczne 10000 można by wyekstrahować do stałej (skrót ctrl + alt + c) i nazwać np. SATISFYING_SALARY
fajno by się to wtedy czytało:) a i potem gdybyśmy użyli tego w paru miejscach i chcieli zmienić tą wartośc wystarczyłoby zrobić to tylko w jednym miejscu:)